src/components/AddImage.js

```diff
@@ -0,0 +1,39 @@
+import Component from './Component.js';
+
+class AddImage extends Component {
+
+    render() {
+        const form = this.renderDOM();
+        const onAdd = this.props.onAdd;
+
+        form.addEventListener('submit', event => {
+            event.preventDefault();
+
+            const formData = new FormData(form);
+
+            const newImage = {
+                title: formData.get('title'),
+                url: formData.get('url'),
+                description: formData.get('description')
```

Comment:
`keywords: ''`

setting keywords to an empty string would bypass this issue, but this is a bandaid to a bigger wound. I would suggest adding another input for keywords and updating things accordingly. As of right now you should not need to change anything about the filter, just the data being collected here.